src/style.css

```
@@ -0,0 +1,126 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  display: flex;
+  width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  display: flex;
+  align-self: start;
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.block-stars {
+  display: flex;
+}
+
+.block-stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.block-stars > :last-child {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.review {
+  display: flex;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-self: stretch;
+  align-items: flex-end;
+  margin-top: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-self: stretch;
+  align-items: center;
+  margin-top: 23px;
+}
+
+.price__text {
+  font-size: 12px;
+  color: #616070;
+}
+
+.price__sum {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+  height: 40px;
+  margin: 16px 0;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: 5px;
+  background-color: #00acdc;
+}
+
+.button {
+  color: #fff;
+  text-decoration: none;
+}
+
+.card__button:hover {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+}
+
+.button:hover {
+  color: #00acdc;
+}
```

```suggestion
.card__button:hover {
  background: #fff;
  border: 1px solid #f3f3f3;
}

.button:hover {
  color: #00acdc;
}
```
Це все мало б бути для одного елемента